ecchronos-binary/src/resources/logback.xml

```diff
@@ -65,6 +69,5 @@
     <root level="DEBUG">
         <appender-ref ref="STDOUT" />
         <appender-ref ref="FILE_INFO" />
-        <appender-ref ref="FILE_DEBUG" />
```

Comment:
Since this debug is removed from default, I would also write that in the documentation (where appropriate). That is, clarify that the info level is the only log level enabled by default and if debug is necessary, describe how to enable it.